FearlessUtils/Classes/Crypto/BIP32KeyFactory.swift

```diff
@@ -0,0 +1,111 @@
+import CommonCrypto
+import IrohaCrypto
+import BigInt
+
+public enum BIP32KeyFactoryError: Error {
+    case invalidChildKey
+}
+
+protocol BIP32KeyFactoryProtocol {
+    func deriveFromSeed(_ seed: Data) throws -> BIP32ExtendedKeypair
+    func createKeypairFrom(_ parentKeypair: BIP32ExtendedKeypair, chaincode: Chaincode) throws -> BIP32ExtendedKeypair
+}
+
+public struct BIP32KeyFactory {
+    private let initialSeed = "Bitcoin seed"
```

Comment:
I think we could make state to prevent from duplication

Reply:
Could you please elaborate more? What is duplicated?

Reply:
Sorry, fixed typos

Reply:
Done